snuba/web/rpc/common/aggregation.py

```diff
@@ -233,9 +233,10 @@ def get_average_sample_rate_column(aggregation: AttributeAggregation) -> Express
         referenced_column=aggregation.label,
         metadata={},
     ).to_alias()
-    return f.avgIf(
-        f.divide(literal(1), sampling_weight_column),
-        get_field_existence_expression(aggregation),
+    field = attribute_key_to_expression(aggregation.key)
+    return f.divide(
+        f.count(field),
```

Comment:
shouldn't this be

sumIf(sign, exists_expression) / sumIf(sign * sampling_weight, exists_expression)

Reply:
also should probably have a test!